metadata-helper/src/config/cli.rs

```diff
@@ -0,0 +1,89 @@
+//! Command-line options for metadata-helper.
+
+use super::options;
+use super::AppSettings;
+use commons::prelude_errors::*;
+use commons::MergeOptions;
+
+/// CLI configuration flags, top-level.
+#[derive(Debug, StructOpt)]
+pub struct CliOptions {
+    /// Verbosity level
+    #[structopt(short = "v", parse(from_occurrences))]
+    pub verbosity: u64,
+
+    /// Path to configuration file
+    #[structopt(short = "c")]
+    pub config_path: Option<String>,
+
+    // Status service options
+    #[structopt(flatten)]
+    pub service: options::ServiceOptions,
+
+    // Main service options
+    #[structopt(flatten)]
+    pub status: options::StatusOptions,
```

Comment:
I'm fuzzy  on details, but `Status service options` for `service` and `Main service options` for `status` seems like the comments may be flipped here?